index.html

```diff
@@ -431,13 +430,18 @@
         immediately after the wake lock is released.
       </div>
       <p>
-        A <a>browsing context</a> is <dfn>requesting the wake lock</dfn> of
-        type <var>type</var> if and only if the following procedure returns
+        A <a>Document</a> is <dfn>requesting the wake lock</dfn> of type
+        <var>type</var> if and only if the following procedure returns
         <code>true</code>:
       </p>
       <ol>
-        <li>Let <var>navigator</var> be the <a>Navigator</a> object associated
-        with this <a>browsing context</a>.
+        <li>Let <var>document</var> be this <a>Document</a> object.
```

Comment:
It's not ambiguous because it refers to the definition above: `A Document is requesting the wake lock of type type if and only if the following procedure returns true:`. Later: `The user agent must acquire the wake lock of type type when all of the following conditions become true:` ... `There is at least one Document that is requesting the wake lock of type type.`. 

About using current settings object I'm not so sure because more than Document object and one Navigator object (through which you actually request wake locks) can be accessible from a browsing context, e.g. those in same-origin child browsing contexts. This is not a problem for Payment Request example which you provided earlier (https://www.w3.org/TR/payment-request/#constructor) because PaymentRequest instances are obtained through constructor which is unambiguously executed in the context of the current settings object.